word-search-page/word-search-page.css

```diff
@@ -0,0 +1,164 @@
+/* THis is general styling for mobile window */
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    outline: none;
+}
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #DFD1E0;
+}
+.whole-page {
+    width: 100%;
+    height: 97%;
+}
+/* This is for the page content to be in the middle */
+.main-container {
+    padding: 25px;
+    width: 100%;
+    height: 100vh;
+    margin: auto;
+}
+/* This is title styling */
+.header {
+    margin-top: 40px;
+    padding: 8px;
+    border-radius: 20px;
+    text-align: center;
+    background-color: #905983;
+    color: #fff;
+    width: 95%;
+    height: 70px;
+    line-height: 60px;
+    margin-right: 10px;
+    font-size: 2.5rem;
+}
+/* This is image styling */
+.image {
+    width: 85%;
+    height: 30%;
+    margin-right: 10px;
+}
+.photo{
+    margin-top: 15px;
+    width: 109%;
+    height: 95%;
+    border-radius: 45px;
+}
```

Comment:
**use this property with an image here to save it's quality  :** 

`object-fit: contain;`